File: Java/PrimeNumber.java
import java.util.Scanner;
class PrimeNumber{
    public static void main(String[] args) {
        boolean flag = true;
        int half = 0;
        System.out.println("Enter a number to find whether it is a prime or Not :-");
        Scanner sc = new Scanner(System.in);
        int number = sc.nextInt();
        half = number/2;
        if(number > 1){
            for(int num = 2;num <= half;num++){
                if(number % num == 0){
                    System.out.println("It is not a Prime number");
                    flag = false;
                    break;
                }
            }
            if(flag){
                System.out.println(number + " is a Prime Number");
            }
        }else{
            System.out.println("It is not a Prime number");
        }
    }
}

File: Python/prime_number.py
num = abs(int(input("Enter a number to check whether it is a prime or not : ")))
checker = 2
if num == 0:
        print("It is not a prime number")
elif num == 1 or num == 2:
        print("It is a prime number")
else:
    while checker < num:
        if num % checker == 0:
            print("It is not a prime number")
            break
        else:
            print("It is a prime number")
            break